test/templates/action-bar/index.njk

```diff
@@ -1,98 +1,16 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
-{% from "../button/component.njk" import button %}
-{% from "./component.njk" import action_bar %}
-{% from "../menu/component.njk" import menu %}
-{% from "./../popover/component.njk" import popover %}
+{% from "./component.njk" import action_bar, action_bar_headlines %}
 {% import "./../utils.njk" as utils %}
 
 
 {% block content %}
-<style media="screen">
-    .fd-action-bar {
-        margin: 0 20px;
-    }
-</style>
-
-{%- set primarybtn %}{{
-    button(
-            { label: 'Button' },
-            modifier={block: [ 'emphasized'] }
-        )
-}}{%- endset %}
-
-{%- set secbtn %}{{
-  button(
-          { label: 'Button' },
-          modifier={ block: [] }
-      )
-}}{%- endset %}
-
-
-{%- set popoverBtn %}
-    {%- set _id = utils.id() %}
-    {%- macro sample_control(properties={}) -%}
-        <button class="fd-button fd-popover__control{{ modifier.control | modifier('popover__control') }}" aria-label="label" aria-expanded="false" aria-haspopup="true" aria-controls="{{ properties.id }}">Popover</button>
-    {%- endmacro %}
-    {{ popover(properties={
-            id: _id,
-            control: sample_control(properties={id: _id}),
-            body: menu(properties={
-            items: [
-                { "label": "Option 1" },
-                { "label": "Option 2" },
-                { "label": "Option 3" },
-                { "label": "Option 4" }
-                ]
-            })
-        }, modifier={ block: ['left']})
-    }}
-{%- endset %}
-
-{%- set popoverBtn2 %}
-    {%- set _id = utils.id() %}
-    {%- macro sample_control(properties={}) -%}
-        <button class="fd-button fd-popover__control{{ modifier.control | modifier('popover__control') }}" aria-label="label" aria-expanded="false" aria-haspopup="true" aria-controls="{{ properties.id }}">Popover</button>
-    {%- endmacro %}
-    {{ popover(properties={
-            id: _id,
-            control: sample_control(properties={id: _id}),
-            body: menu(properties={
-            items: [
-                { "label": "Option 1" },
-                { "label": "Option 2" },
-                { "label": "Option 3" },
-                { "label": "Option 4" }
-                ]
-            })
-        }, modifier={ block: ['left']})
-    }}
-{%- endset %}
+{%- macro render_txt_button() %}<div class="test-btn-placeholder">Button</div>{%- endmacro %}
+{%- macro render_icon_button() %}<div class="test-btn-placeholder sap-icon--overflow"></div>{%- endmacro %}
```

Comment:
Can we replace `test-btn-placeholder` with `btn-playground`?  Also, we shouldn't be using the class `sap-icon--overflow` in the playground.